docs/content/code-generator.md

```diff
@@ -66,25 +66,12 @@ After post processing, a code generation job is ran on each C# object that repre
 The process of editing the code generator is simple:
 
 1. Make the code and/or template changes.
-2. Run `spatial codegen` to see your changes in action! The code generator will automatically be rebuilt.
+2. Run `ci/codegen.sh` to see your changes in action! The code generator will automatically be rebuilt.
 
 
 #### Spatial Codegen configuration
 
-`spatial codegen` uses the [worker build configuration](https://docs.improbable.io/reference/latest/shared/worker-configuration/worker-build#using-custom-build-scripts) to determine what to run. In both `spatialos.unity.client.build.json` and `spatialos.unity.gamelogic.build.json` there is a step that runs the custom code generator described above.
-
-```json
-{
-  "name": "GDK Codegen",
-  "command": "../../bin/code_generator/GdkCodeGenerator.exe",
-  "arguments": [
-      "--json-dir=./improbable/json/usr",
-      "--output-dir=./Assets/Generated/"
-  ]
-}
-```
-
-If any changes you make change the input parameters or add new ones, you should add them to those two json files to ensure that `spatial codegen` runs correctly!
+If any changes you make change the input parameters or add new ones, you should add them `ci/codegen.sh` so it runs correctly!
```

Comment:
you should add them ***to*** `ci/codegen.sh`**

Reply:
Done